VestingInstance.sol

```diff
@@ -0,0 +1,273 @@
+pragma solidity ^0.5.3;
+
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "../common/UsingRegistry.sol";
+
+contract VestingSchedule is Ownable, UsingRegistry {
+    // solhint-disable not-rely-on-time
+
+    modifier onlyRevoker() {
+      require(msg.sender == _revoker, "sender must be the vesting revoker");
+      _;
+    }
+
+    modifier onlyBeneficiary() {
+      require(msg.sender == _revoker, "sender must be the vesting beneficiary");
+      _;
+    }
+
+    using SafeMath for uint256;
+
+    event VestingGoldLocked(uint256 amount, uint256 timestamp);
+    event VestingGoldUnlocked(uint256 amount, uint256 timestamp);
+    event VestingGoldRelocked(uint256 index, uint256 timestamp);
+    event VestingGoldWithdrawn(uint256 index, uint256 timestamp);
+    event VestingAccountVoterAuthorized(address authorizer, address voter, uint256 timestamp);
+    event VestingAccountValidatorAuthorized(address authorizer, address validator, uint256 timestamp);
+    event VestingWithdrawn(address beneficiary, uint256 amount, uint256 timestamp);
+    event VestingRevoked(address revoker, address refundDestination, uint256 refundDestinationAmount, uint256 timestamp);
+
+    // total that is to be vested
+    uint256 public _vestingAmount;
```

Comment:
I do not think it is good idea to have a public variable named with `_` or `__`. If you've named them this way because of conflict of names with the constructor, I'd much rather to change the names in the constructor. Keep in mind that this public modifier will generate a public getter and `contract._vestingAmount()` seems a bit misleading as name for a getter function